### ISSUE TITLE
Checking for errors for downloads

### ISSUE THREAD

**Issue by robertmryan** [Alamofire/Alamofire#1705]
In the README, it says you can check for download errors like so:

```swift
Alamofire.download(urlString, to: destination).response { response in
    print(response)

    if response.result.isSuccess, let imagePath = response.destinationURL?.path {
        let image = UIImage(contentsOfFile: imagePath)
    }
}
```

But `response`, a `DefaultDownloadResponse`, doesn't have a `result` property. I guess you can just check to see if `error` was `nil` or not:

```swift
Alamofire.download(urlString, to: destination).response { response in
    guard response.error == nil else {
        print(response.error!)
        return
    }

    if let imagePath = response.destinationURL?.path {
        let image = UIImage(contentsOfFile: imagePath)
    }
}
```

I'd suggest we either add the `result` property, or update the README.

**Comment by jshier** [comment]
This will be addressed in #1722.

**Comment by jshier** [comment]
This has been merged into `master`.
